notebooks/official/training/multi_node_ddp_nccl_vertex_training_with_custom_container.ipynb

```diff
@@ -1184,9 +1039,16 @@
         "id": "73c92c9298e9"
       },
       "source": [
-        "### Create a Vertex AI Tensorboard instance\n",
+        "## Create a Vertex AI Tensorboard instance\n",
+        "\n",
+        "Set a display name for your Tensorboard instance below and run the create command.\n",
+        "\n",
+        "**Note:**  To use an existing Vertex AI Tensorboard instance, you can use the following method.\n",
```

Comment:
remove "you can":
-  "To use an existing Vertex AI Tensorboard instance, use the following method."